src/main/java/org/sagebionetworks/bridge/RequestContext.java

```diff
@@ -45,7 +45,7 @@ public static RequestContext updateFromSession(UserSession session, SponsorServi
         builder.withCallerAppId(session.getAppId());
 
         StudyParticipant participant = session.getParticipant();
-        if (participant.getOrgMembership() != null && !BridgeUtils.isInRole(participant.getRoles(), ADMIN)) {
```

Comment:
We don't want to engage in shenanigans here...if the role prevents the check, then check for the role, don't just make the list of studies an empty list. The empty list checks are going away.